src/main/kotlin/skytils/skytilsmod/features/impl/handlers/SpamHider.kt

```diff
@@ -380,6 +417,42 @@ class SpamHider {
                 }
             }
 
+            // Sea creature catch
+            if (SEA_CREATURES.contains(unformatted)) {
+                when (Skytils.config.scCatchHider) {
+                    1, 2 -> cancelChatPacket(event, Skytils.config.scCatchHider == 2)
+                    else -> {
+                    }
+                }
+            }
+
+            // Legendary sea creature catch
+            if (LEGENDARY_SEA_CREATURES.contains(unformatted)) {
+                when (Skytils.config.legendaryScCatchHider) {
+                    1, 2 -> cancelChatPacket(event, Skytils.config.legendaryScCatchHider == 2)
+                    else -> {
+                    }
+                }
+            }
+
+            // Good catch
+            if (unformatted.startsWith("GOOD CATCH! You found")) {
+                when (Skytils.config.goodTreasureHider) {
+                    1, 2 -> cancelChatPacket(event, Skytils.config.goodTreasureHider == 2)
+                    else -> {
+                    }
+                }
+            }
+
+            // Great catch
```

Comment:
This one too.

Reply:
Should be fine because it is impossible for another player to send a message that starts with this